xwiki-platform-core/xwiki-platform-notifications/xwiki-platform-notifications-ui/src/main/resources/XWiki/Notifications/Code/Translations.xml

```diff
@@ -45,13 +45,13 @@ platform.core.profile.category.notifications=Notifications
 notifications.filters.cancel=Cancel
 notifications.filters.submit=Save
 notifications.filters.type.label=Filter type:
-notifications.filters.type.hint=How the filter should act towards an event in the given location
+notifications.filters.type.hint=How should the filter respond to an event in the given location?
 notifications.filters.location.label=Filter scope:
-notifications.filters.location.hint=What part of the wiki should be concerned by the filter?
+notifications.filters.location.hint=About which part of the wiki should the filter be concerned?
```

Comment:
Not sure about this one, are you sure it's the same meaning ? The previous one was a very French construction so most probably not right but the new version looks very different.

Reply:
Yes it looks very different but it's grammatically correct and conveys the right meaning AFAIK